bittensor/_wallet/wallet_impl.py

```diff
@@ -81,48 +81,84 @@ def __str__(self):
     def __repr__(self):
         return self.__str__()
 
-    def neuron(self, netuid: int) -> SimpleNamespace:
-        return self.get_neuron(netuid = netuid)
+    def neuron(self, netuid: int) -> Optional['bittensor.NeuronInfo']:
+        return self.get_neuron(netuid=netuid)
 
-    def trust(self, netuid: int) -> SimpleNamespace:
-        return self.get_neuron(netuid = netuid).trust
+    def trust(self, netuid: int) -> Optional[float]:
+        neuron = self.get_neuron(netuid=netuid)
+        if neuron is None:
+            return None
+        return neuron.trust
+
+    def validator_trust(self, netuid: int) -> Optional[float]:
+        neuron = self.get_neuron(netuid=netuid)
+        if neuron is None:
+            return None
+        return neuron.validator_trust
```

Comment:
@camfairchild wallet_impl.py field methods needed type hint corrections and handling of possible None neuron.